integration/spark/app/build.gradle

```diff
@@ -94,10 +94,12 @@ dependencies {
     compileOnly("org.apache.spark:spark-hive_${scala}:${spark}")
     compileOnly("org.apache.spark:spark-sql-kafka-0-10_${scala}:${spark}")
 
+
     testImplementation("org.apache.spark:spark-core_${scala}:${spark}")
     testImplementation("org.apache.spark:spark-sql_${scala}:${spark}")
     testImplementation("org.apache.spark:spark-hive_${scala}:${spark}")
     testImplementation("org.apache.spark:spark-sql-kafka-0-10_${scala}:${spark}")
+    testImplementation("org.mongodb.spark:mongo-spark-connector_${scala}:10.1.1")
```

Comment:
Would it make sense to have integration test in docker that downloads `mongo-spark-connector` when being run? 
I know we're doing a lot of  `testImplementation` dependencies in openlineage-spark repo, but the problem with this are: 
 * We're polluting test classpath with classes that will be present in test runtime but won't be available at the runtime. 
 * We're limiting the possibility to run tests against different mongo connector versions in the future which would be possible for docker tests. 

What do you think?

Reply:
Make sense I can put the test in a different file and run spark in container.

Reply:
Great test that proves the feature is working. thank you